table/temptable/ddl.go

```diff
@@ -180,6 +180,12 @@ func newTemporaryTableFromTableInfo(sctx sessionctx.Context, tbInfo *model.Table
 	alloc := autoid.NewAllocatorFromTempTblInfo(tbInfo)
 	allocs := make([]autoid.Allocator, 0, 1)
 	if alloc != nil {
+		// Rebase the allocator if the base is specified.
+		if tbInfo.AutoIncID > 1 {
```

Comment:
Why not moving these codes to `NewAllocatorFromTempTblInfo` ?

Reply:
I want to make `NewAllocatorFromTempTblInfo` consistent with `NewAllocatorsFromTblInfo`: `NewAllocatorsFromTblInfo` doesn't rebase the allocator, and the callers rebase the allocators.

Reply:
I think it's also fine to break the consistency. I'll do it.